src/bust.py

```diff
@@ -13,6 +13,7 @@
     ClientException,
     Embed,
     FFmpegPCMAudio,
+    FFmpegOpusAudio,
```

Comment:
I think this import is unsorted, run scripts-dev/lint.sh to sort, format, and run linters that we use for github automation